### ISSUE TITLE
Is it possible to import multiple ip or domain names in the function of Split Tunnel at one time?

### ISSUE THREAD

**Issue by YiGeRen-yl** [Windscribe/Desktop-App#71]
Is it possible to import multiple ip or domain names in the function of Split Tunnel at one time?

**Comment by YiGeRen-yl** [comment]
Hello, the rules of the domain name will not affect the addition of IP, whether it is possible to add IP addresses in batches,this reduces time cost for user.

**Comment by bufce2010** [comment]
is it effective to add IPs with notation like this 10.11.12.0/24 ?

**Comment by bufce2010** [comment]
> Is it possible to import multiple ip or domain names in the function of Split Tunnel at one time?

this is my method of adding multiple IPs:
use "Macro Recorder" and record the click of your keybaord : Ctrl+C to copy , alt+tab to switch between windows, Ctrl+V to paste, click Enter to add the IP
This is a SS for the window properties to adjust:
![image](https://github.com/user-attachments/assets/dbb8f0d7-f758-454c-9a8a-440445e88c11)

adjust the delay time for all stpes by clicking Ctrl+F and chose delay\timeout, then add 50 or 100 milliseconds only as a delay
and set the macro app to repeat that for as much as you need, like this SS:
![image](https://github.com/user-attachments/assets/78f1a8ea-2e7b-4eb3-960d-9ef46f0c5285)

**Comment by bernerdad** [comment]
> > Hi. Unfortunately that is not currently possible. We may add a mask feature in the near future, where you can specify something like *.google.com for inclusion/exclusion.
> 
> is it effective to add IPs with notation like this 10.11.12.0/24 ?

Yes, you can use that notation.  Please let us know by opening a new issue if you find it does not work.
